src/components/OurCourses/index.js

```diff
@@ -0,0 +1,79 @@
+import { Typography, Grid, useMediaQuery } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+import Carousel from "react-multi-carousel";
+
+import useStyles from "./useStyles";
+
+import OurCourseCard from "@/pesayetu/components/InsightCard";
+import Section from "@/pesayetu/components/Section";
+import "react-multi-carousel/lib/styles.css";
+
+const responsive = {
+  desktop: {
+    breakpoint: {
+      max: 3000,
+      min: 1280,
+    },
+    items: 3,
+  },
+  tablet: {
+    breakpoint: { max: 1279, min: 768 },
+    items: 2,
+  },
+  mobile: {
+    breakpoint: { max: 767, min: 0 },
+    items: 1,
+  },
+};
+
+const OurCourses = ({ title, items, ...props }) => {
+  const classes = useStyles(props);
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up("lg"));
+
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        {title && (
+          <Typography variant="h3" className={classes.title}>
+            {title}
+          </Typography>
+        )}
+        <Carousel
+          swipeable
+          responsive={responsive}
+          arrows={false}
+          renderDotsOutside
+          showDots={!isDesktop}
+          dotListClass={classes.dots}
+        >
+          {items?.map((item) => (
+            <Grid item key={item.title}>
+              <OurCourseCard {...item} />
```

Comment:
Just like you did for AboutTeam @gertie-sheshe? Why aren't we just importing `InsightCard` and use it here?

Reply:
It is the InsightCard, I just changed the import name 😅 @kilemensi

Reply:
Cool @gertie-sheshe ... then why is [`OurCourseCard`](https://github.com/CodeForAfrica/PesaYetu/pull/55/files#diff-47c6b9a0c492fa3047847e3a077bdda3156a05a53712ae1e733c4b7b04cb59c6R1) still around?

Reply:
ooh..yeah, you're right. Fixed